gpm_api/tests/io/test_checks.py

```diff
@@ -337,9 +337,9 @@ def test_check_time() -> None:
     assert res == datetime.datetime(2014, 12, 31)
 
     # Test a object of datetime64[ns] casts to datetime64[ms]
-    res = checks.check_time(np.datetime64("2014-12-31T12:30:30.934549845", "ns"))
```

Comment:
The function `check_time` here must accept (and was accepting) `np.datetime64 `with `ns` resolution and cast it to seconds accuracy within the function !

Reply:
I've re-added this test in, and test it casts to seconds